Packs/Asset/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Asset",
+    "description": "Base pack for any packs using Assets.",
```

Comment:
@yaron-libman @kirbles19 
Hi we decided to create a new pack with incident fields just for assets
It contains only the previous incident fields from before
Please review if needed